packages/mobile/src/navigator/DrawerTopBar.tsx

```diff
@@ -1,46 +1,60 @@
-import colorsV2 from '@celo/react-components/styles/colors.v2'
-import { elevationShadowStyle } from '@celo/react-components/styles/styles'
+import HamburgerIcon from '@celo/react-components/icons/Hamburger'
+import colors from '@celo/react-components/styles/colors.v2'
 import { useNavigation } from '@react-navigation/native'
 import * as React from 'react'
 import { StyleSheet, TouchableOpacity, View } from 'react-native'
-import Logo from 'src/icons/Logo.v2'
-import Hamburger from 'src/navigator/Hamburger'
+import Animated, { cond, greaterThan } from 'react-native-reanimated'
 
-const DrawerTopBar = () => {
+interface Props {
+  middleElement?: React.ReactNode
+  scrollPosition?: Animated.Value<number>
+}
+
+function DrawerTopBar({ middleElement, scrollPosition }: Props) {
   const navigation = useNavigation()
+  const viewStyle = React.useMemo(
+    () => ({
+      ...styles.container,
+      borderBottomColor: colors.gray2,
+      borderBottomWidth: cond(greaterThan(scrollPosition ?? new Animated.Value(0), 0), 1, 0),
+    }),
+    [scrollPosition]
+  )
 
   return (
-    <View style={styles.container}>
+    <Animated.View style={viewStyle}>
       {/*
       // @ts-ignore Only used in a drawer */}
       <TouchableOpacity style={styles.hamburger} onPress={navigation.toggleDrawer}>
-        <Hamburger />
+        <HamburgerIcon />
       </TouchableOpacity>
-      <View style={styles.logo}>
-        <Logo />
-      </View>
+      {middleElement}
       <View style={styles.spacer} />
-    </View>
+    </Animated.View>
   )
 }
 
+DrawerTopBar.defaultProps = {
+  showLogo: true,
+}
+
 const styles = StyleSheet.create({
   container: {
     flexDirection: 'row',
     height: 62,
-    backgroundColor: colorsV2.white,
-    alignItems: 'flex-end',
+    alignItems: 'center',
+    backgroundColor: colors.background,
     justifyContent: 'space-between',
-    ...elevationShadowStyle(1),
+  },
+  withBorderBottom: {
```

Comment:
It doesnt look like this styling is being applied anywhere

Reply:
good catch, will address it as a part of my next PR 👍